src/SourceBuild/content/eng/tools/init-build.proj

```diff
@@ -15,7 +15,7 @@
       UnpackTarballs;
       BuildUnifiedBuildTasks;
       BuildMSBuildSdkResolver;
-      BuildSdkArchiveDiff;
+      BuildUnifiedBuildValidationTests;
```

Comment:
Now that this is test project this doesn't belong here anymore but we can do that in a follow-up.

Reply:
Will it automatically get restored by arcade as a test project? Or can we assume it will never be run in an offline build?

Reply:
Yes, it will get restored same as the build.proj and the repo-projects/ .proj files. But as mentioned, it might be better to do that in a follow-up.

Reply:
Looks like building this before the sdk version is determined, and the runtime config values aren't being set, so it'll need to be done here.